sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/models/ModelBridgeInternal.java

```diff
@@ -104,16 +103,16 @@ public static CosmosAsyncPermissionResponse createCosmosAsyncPermissionResponse(
     }
 
     @Warning(value = INTERNAL_USE_ONLY_WARNING)
-    public static CosmosAsyncStoredProcedureResponse createCosmosAsyncStoredProcedureResponse(ResourceResponse<StoredProcedure> response,
-                                                                                              CosmosAsyncContainer cosmosContainer) {
-        return new CosmosAsyncStoredProcedureResponse(response, cosmosContainer);
+    public static CosmosStoredProcedureResponse createCosmosStoredProcedureResponse(ResourceResponse<StoredProcedure> response,
+                                                                                         CosmosAsyncContainer cosmosContainer) {
+        return new CosmosStoredProcedureResponse(response);
     }
 
     @Warning(value = INTERNAL_USE_ONLY_WARNING)
-    public static CosmosAsyncStoredProcedureResponse createCosmosAsyncStoredProcedureResponse(StoredProcedureResponse response,
-                                                                                              CosmosAsyncContainer cosmosContainer,
-                                                                                              String storedProcedureId) {
-        return new CosmosAsyncStoredProcedureResponse(response, cosmosContainer, storedProcedureId);
+    public static CosmosStoredProcedureResponse createCosmosStoredProcedureResponse(StoredProcedureResponse response,
+                                                                                         CosmosAsyncContainer cosmosContainer,
```

Comment:
the args indentation should be the same.

Reply:
Done.